### ISSUE TITLE
Rename tabs

### ISSUE THREAD

**Issue by subhead** [felixse/FluentTerminal#141]
Hi,

is it possible to add a feature for renaming tabs? At the moment i need to place my mouse over the tab and read the hover text to get clue ion which terminal i work at the moment.

**Comment by skipmeister123** [comment]
That can be done via environment variables.
--
Hans Kokx

**Comment by subhead** [comment]
yes i know but i find it more convenient to rename the tabs over the gui.

**Comment by rafd123** [comment]
FWIW, as commented on #154,  here's how I do it: 

**cmd.exe**
```
title foo
```

**PowerShell**
```
[console]::Title = 'foo'
```

**bash/WSL**
```
echo -e '\033]2;foo\007'
```

Not suggesting this enhancement shouldn't be implemented, just putting this here as a tip since:
- It tends to work for all terminals (even cmd.exe)
- When wrapped in a shell function, renaming tabs super convenient since you don't have to lift your hands from the keyboard

**Comment by ericcornelissen** [comment]
It seems this issue was resolved with d4aadd7 @felixse?

However, I'm having an issue with the renaming where `npm` (and npm-based commands such as `gulp`) overrides the name and restores the default. For example when I change the title of a tab to "Foo", running `npm install` changes the title to `npm`, and when `npm install` finished the name is restored to "powershell" or "cmd".
